### PR TITLE
new datetime renderer

### PR DESCRIPTION
**Pull request by mirackle-spb**
Sometimes we need to render ISO time. This is datetime renderer with ISO8601 format.

### CODE REVIEW

File: producer/proto/render.go [hunk 1]
```diff
@@ -115,6 +118,13 @@ func IPRenderer(msg *ProtoProducerMessage, fieldName string, data interface{}) i
 	return NilRenderer(msg, fieldName, data)
 }
 
+func DateTimeRenderer(msg *ProtoProducerMessage, fieldName string, data interface{}) interface{} {
+	if dataC, ok := data.(uint64); ok {
+		return time.UnixMicro(int64(dataC / 1000)).Format(time.RFC3339Nano)
```
Comment (lspgn) [inline producer/proto/render.go:123]: Needs to support nanoseconds. Not just micro.

**Comment by lspgn** [comment]
Thank you for this suggestion. I will have a look.
The renderer way seems to be the best, but I may rename/add additional time formats.

File: producer/proto/render.go (hunk 1 above)
Comment (mirackle-spb) [reply producer/proto/render.go:123]: ISO time format does not support nano

Comment (lspgn) [reply producer/proto/render.go:125]: I'm ok adding RFC3339Nano which supports it.
Please provide reasons that you need to truncate to Micro.

**Comment by lspgn** [comment]
I don't think localdatetime should be added.
Again please provide use case.

**Comment by lspgn** [comment]
Closing due to no reply.

**Pull request closed without merging**